test/Compatibility/tuple_arguments.swift

```diff
@@ -1478,3 +1478,10 @@ let pages3: MutableProperty<(data: DataSourcePage<Int>, totalCount: Int)> = Muta
     data: DataSourcePage<Int>.notLoaded,
     totalCount: 0
 ))
+
+// rdar://problem/32301091 - [SE-0110] causes errors when passing a closure with a single underscore to a block accepting multiple parameters
```

Comment:
To make it clear, I'd put a comment that for Swift 3 we accept this, like you do below. Otherwise "[SE-0110] causes errors" makes it seem like we expect an error here with this test case. This is a huge file so going back to look at the -swift-version used here is a bit of extra work, and in the future not everyone will remember what SE-0110 was (or perhaps they will not even have seen it).